index.d.ts

```diff
@@ -1,5 +1,12 @@
 export type Property<T> = string | ((element: T) => unknown) | Array<string | ((element: T) => unknown)>;
 
+interface Options {
+	/** A locale string or array of locale strings that contain one or more language or locale tags. If you include more than one locale string, list them in descending order of priority so that the first entry is the preferred locale. If you omit this parameter, the default locale of the JavaScript runtime is used. This parameter must conform to BCP 47 standards; see the Intl.Collator object for details. */
```

Comment:
```suggestion
	/**
	A locale string or array of locale strings that contain one or more language or locale tags.
	
	If you include more than one locale string, list them in descending order of priority so that the first entry is the preferred locale.
	
	If you omit this parameter, the default locale of the JavaScript runtime is used. 
	
	This parameter must conform to BCP 47 standards; see the Intl.Collator object for details.
	*/
```